src/main/java/org/folio/circulation/domain/policy/LoanPolicy.java

```diff
@@ -155,7 +159,11 @@ public static LoanPolicy unknown(String id) {
 
         isRenewalWithHoldRequest = true;
       }
-
+      if (hasDeclaredLostItem(loan)) {
+        errors.add(itemByIdValidationError(DECLARED_LOST_ITEM_RENEWED_ERROR,
+          loan.getItemId()));
+        return failedValidation(errors);
+      }
```

Comment:
Why is this validation being conducted here? How does the loan policy impact the decision to disallow a renewal when an item is declared lost?

Reply:
The different types of validation which are connected with renew process are placed here. 
It seems to me they should be in one place. I agree that the loan policy class doesn't suitable place for the declared lost validation. 
Along with that I suggest leaving the validation here or moving out the renew method from Loan Policy completely?

Reply:
> It seems to me they should be in one place. I agree that the loan policy class doesn't suitable place for the declared lost validation.

I think there might be a distinction between validation that is universal to all renewal and some that is based upon specific policies.

> Along with that I suggest leaving the validation here or moving out the renew method from Loan Policy completely?

I think much of it probably needs to come out of the loan policy. Is this is the change to do that work?

Reply:
Changed